source/install/install-rhel-71-postgresql.rst

```diff
@@ -19,19 +19,19 @@ Installing PostgreSQL Database
 
 5. Initialize the database.
 
-  ``sudo service postgresql initdb``
+  ``sudo /usr/pgsql-9.4/bin/postgresql94-setup initdb``
 
 6. Set PostgreSQL to start on boot.
 
-  ``sudo systemctl enable postgresql``
+  ``sudo systemctl enable postgresql-9.4``
 
 7. Start the PostgreSQL server.
 
-  ``sudo systemctl start postgresql``
+  ``sudo systemctl start postgresql-9.4``
 
 8. Switch to the *postgres* Linux user account that was created during the installation.
 
-  ``sudo --login --user postgres``
+  ``sudo su postgres``
```

Comment:
Ok. `--login` and `--user` work with the latest version of su, but that might not be on RHEL 7.1

The equivalent short options are `-i` and `-u`. The `-i` option is different than the `-s` option though... I think it changes you to the postgres user directory, among other things.

If you can confirm that `su` works and that `iu` is not needed, then we can keep `su`.